actors/builtin/miner/miner_actor.go

```diff
@@ -1443,7 +1425,9 @@ func (a Actor) WithdrawBalance(rt Runtime, params *WithdrawBalanceParams) *adt.E
 		rt.Abortf(exitcode.ErrIllegalArgument, "negative fund requested for withdrawal: %s", params.AmountRequested)
 	}
 	var info *MinerInfo
-	newlyVestedAmount := rt.State().Transaction(&st, func() interface{} {
+	newlyVested := big.Zero()
```

Comment:
I'd leave this (and the one above) zero initialized. That way, if we end up adding a stray colon below and fail to actually initialize this, this function will always panic.

Reply:
Hmm. The reason I so conscientiously init these is that we had such panics on devnets in the past (though on an occasional code path).

Reply:
I figured an obvious panic is better than a silent zero.